balancer/frontend-v2
# Move token addresses to the tokens config file

# Description

Moves all the token addresses from the contracts file to the tokens file. This is a more correct place for them and they aren't statically defined so the list can be custom to the network instead of having so many empty strings on networks that don't support LSD tokens. 

I also changed a few places referencing "Weth" when really they meant "Wrapped Native Asset". 

## Type of change


## How should this be tested?

- Ensure places involving stETH / stMATIC work as expected. 
- Ensure SOR and other things that might be affected by this work correctly. 

## Checklist:
